### PR TITLE
Proxy domain support

### PR DESCRIPTION
**Pull request by psquickitjayant**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -57,6 +57,36 @@ Send your custom tags to Loggly by setting the `tag` property.
...
+You can proxy the requests from your own domain if the script or its requests are blocked by Ad blockers. To do this, you need to perform following steps
+
+Set `useProxyDomain` property to true
+
+```Javascript
+_LTracker.push({
+  'logglyKey': '8c518f97-e3e0-4bfb-a8ed-582d084a5289',
+  'sendConsoleErrors' : true,
+  'tag' : 'javascript-logs',
+  'useDomainProxy' : true
```
Comment (mostlyjason) [inline README.md]: Why not just use the domain proxy if it's not null? Wouldn't it be simpler to have one variable instead of two?

+_LTracker.push({
+  'logglyKey': '8c518f97-e3e0-4bfb-a8ed-582d084a5289',
+  'sendConsoleErrors' : true,
+  'tag' : 'javascript-logs',
+  'domainProxy' : 'loggly.mydomain.com'
+});

Comment (psquickitjayant) [inline README.md]: Are we supporting them to use any domain? We are currently appending /loggly to their current domain. Like

`inputUrl = window.location.host + '/loggly' + '/input/token...;`

Also, we have give the example for the same in server configuration.

Comment (mostlyjason) [inline README.md]: Ahh thanks for explaining! That seems fine too. If someone wants to add other domains they can do it in another PR.
